src/processing/script/header.ts

```diff
@@ -20,7 +21,7 @@ const HEADER: string = `
     if (typeof window !== 'undefined' && window){
         window['${INTERNAL_PROPS.processDomMethodName}'] && window['${INTERNAL_PROPS.processDomMethodName}']();
 
-        if (window.${INSTRUCTION.getProperty} && typeof ${INSTRUCTION.getProperty} === 'undefined') {
+        if (window.${INSTRUCTION.getProperty} && typeof ${INSTRUCTION.getProperty} === 'undefined')
```

Comment:
For which cases does it necessary?

Reply:
https://github.com/DevExpress/testcafe-hammerhead/issues/2404